DEVNOTES.md

```diff
@@ -1,10 +1,10 @@
 # Local Development
 
-1. We use [node@16](https://github.com/nvm-sh/nvm#installing-and-updating):
+1. We use [node@22.6.0](https://github.com/nvm-sh/nvm#installing-and-updating):
 
 ```
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
-nvm install 16
+nvm install 22.6.0
```

Comment:
I'd love it if we could tie this to what's defined in `Dockerfile`, but that's definitely out of scope here.